reversed_func.py

```diff
@@ -0,0 +1,21 @@
+def reversed_func(text: str) -> str:
+
+    reversed_text = []
+    letter_list = [letter for word in reversed(text.split()) for letter in word if letter.isalpha()]
```

Comment:
I disagree with double for loop in list comprehension. It's too much.  We have to balance between fewer lines of code and the readability of the code. Two and more for-loops in a single list comprehension is too much.

Let's make one regular for-loop that iterates over all words in the initial text and inside make a list comprehension that collects all the letters and a second loop that creates reversed string with symbols and digits.

isalpha is good approach